src/api/c/device.cpp

```diff
@@ -30,6 +30,24 @@ af_err af_set_backend(const af_backend bknd) {
     return AF_SUCCESS;
 }
 
+af_err af_add_backend_library(const char *lib_path) {
+    try {
+        // what to do here? not supposed to be called outside of the unified backend
```

Comment:
Opinions on what I should do here? Should this throw an exception instead if used outside of the unified backend?

Reply:
Umar discussed this with me earlier. This code is unnecessary in `src/api/c` because only the unified backend uses this functions, but without this code, the linker complains when building standalone backend libraries (e.g. afcpu) because the C++ API calls these C API functions. Thus, we decided to not include this in the C++ API anymore in order to omit these empty functions in `src/api/c`. Of course, these functions will remain in `src/api/unified`.